LondonBritishLibrary/orient/BLorient768.xml

```diff
@@ -31,21 +31,391 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+
+<msItem xml:id="ms_i1.1">
+<locus from="4r"/>
+<title xml:lang="en" type="complete">Moses of Scete</title>
+<incipit xml:lang="gez">ቀዳሚ፡ ዜና፡ ዘ፩እመነኮሳት፡ ዘገዳመ፡ ሲሐት፡ ዘስሙ፡ ሙሴ።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="13r"/>
+<title xml:lang="gez" type="complete">ካልኣይ፡ ዜና፡ ዘ፩እምቅዱሳን፡ ኅዱራነ፡ ገዳም፡</title>
+<incipit xml:lang="gez">ኮነ፡ አመ፡ አዝማነ፡ አብ፡ ቅዱስ፡ ወክቡር፡ አባ፡ ብንያሚን፡ ሊቀ፡ ጳጳሳት፡ ዘሀገር፡ ዐባይ፡ እለእስክንድርያ። ወ፭አህጉር፡ ካልኣን፡ ኢትዮጵያ፡ ወኖባ።</incipit>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="18r"/>
+<title xml:lang="gez" type="complete">ሣልስ፡ ዜና፡ ዘጽሙድ፡ ዘኮነ፡ ውስተ፡ ገዳም፡ ዘይትበሀል፡ ላቲ፡ አክሊማጥስ፡ ወይእቲ፡ ገዳመ፡ እልፍዩም፡ ይእዜ።</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="21r"/>
+<title xml:lang="gez" type="complete">ራብዕ፡ ዜና፡ ኮነ፡ ኦአኃውየ፡ ወፍቁራንየ፡ ምእመናነ፡ ክርስቶስ፡ ውስተ፡ ዛቲ፡ ሀገር፡ ዘይትበሀል፡ ላቲ፡ አፍራቅያ፡ ብእሲ፡ ባዕል፡
+ብዙኃ፡ ንዋይ፡ ፈድፋደ።</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus from="23r"/>
+<title xml:lang="gez" type="complete">ኃምስ፡ ዜና፡ ዘ፩እምቅዱሳን።</title>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.6">
+<locus from="25r"/>
+<title xml:lang="gez" type="complete">ሳድስ፡ ዜና፡</title>
+<incipit xml:lang="gez">ኮነ፡ ኦአኃውየ፡ ምእመናኒሁ፡ ለክርስቶስ፡ ያስተርክብክሙ፡ እግዚአብሔር፡ ለፈቃዱ፡ ወይምራኅክሙ፡ ኀበ፡ ሥምረቱ፡ ውስተ፡ ምኔት፡
+ዘይትአመር፡ ምኔተ፡ አባ፡ ጳኵሚስ፡ ዘውስተ፡ አድያመ፡ ፅዒድ፡ መነኮስ፡ ቅዱስ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.7">
+<locus from="29v"/>
+<title xml:lang="gez" type="complete">ሳብዕ፡ ዜና፡ ዘ፩እምቅዱሳን።</title>
+<incipit xml:lang="gez">ኮነ፡ ውስተ፡ ምኔት፡ እምኔታተ፡ መነኮሳት፡ እምአድያመ፡ ሀገር፡ ዘትሰመይ፡ ሳን፡ ቅዱስ፡ ዘይሰመይ፡ ስሙ፡ ዩሴፍ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.8">
+<locus from="34r"/>
+<title xml:lang="gez" type="complete">ሳምን፡ ዜና፡</title>
+<incipit xml:lang="gez">ኮነ፡ ኦአኃውየ፡ ውስተ፡ ደብር፡ ሠርቃዊ፡ ዘእምድረ፡ ግብጽ፡ ዘቅሩብ፡ እምባሕረ፡ ኤርትራ፡ ጽሙድ፡ ዘይነብር፡ ውስቴቱ፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.9">
+<locus from="38r"/>
+<title xml:lang="en" type="complete">Anecdote narrated by the patriarch ʾabbā Benjamin</title>
+<incipit xml:lang="gez">ይቤ፡ አቡነ፡ አብ፡ ሊቀ፡ ጳጳሳት፡ አባ፡ ብንያሚ፡</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.10">
+<locus from="42r"/>
+<title xml:lang="en" type="complete">Anecdote narrated by ʾabbā Victor, abbat of the convent of Māhw or
+ʾel-Ḥabṭu</title>
+<incipit xml:lang="gez">ዘ፩እምቅዱሳን፡ እምዘ፡ ጸሐፎ፡ አብ፡ ቅዱስ፡ አባ፡ ፊቅጦር፡ ሊቀ፡ ምኔት፡ ዘእልሐብጡ፡ ወይትዓወቅ፡ ምኔተ፡ ማኅው፡ ዘውስተ፡
+ምድረ፡ ሀገረ፡ እለእስክንድርያ።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.11">
+<locus from="45r"/>
+<title xml:lang="en" type="complete">Another anecdote narrated by ʾabbā Victor</title>
+<incipit xml:lang="gez">ይቤ፡ ፊቅጦር፡ ጸሓፊ፡ እንዘ፡ አነ፡ ፩ዕለተ፡ ውስተ፡ ማኅበረ፡ አኃው፡ ውስተ፡ ዝንቱ፡ ምኔት።</incipit>
+<textLang mainLang="gez"></textLang>
+</msItem>
+
+<msItem xml:id="ms_i1.12">
+<locus from="48r"/>
+<title xml:lang="en" type="complete">Another</title>
```

Comment:
Another? I think it would be better to add something because it is very understandable.

Reply:
Same for the items below

Reply:
It's all Wright's fault :(